Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -104,18 +47,23 @@ namespace O3DE::ProjectManager
             {
                 emit GoToPreviousScreenRequest();
             });
-        vLayout->addWidget(m_header);
+        screenLayout->addWidget(m_header);
 
-        m_tabWidget = new TabWidget(this);
-        m_tabWidget->setContentsMargins(0, 0, 0, 0);
         
-        m_tabWidget->setObjectName("createGemTab");
-        m_tabWidget->addTab(CreateGemSetupScrollArea(), tr("1. Gem Setup"));
-        m_tabWidget->addTab(CreateGemDetailsScrollArea(), tr("2. Gem Details"));
-        m_tabWidget->addTab(CreateGemCreatorScrollArea(), tr("3. Creator Details"));
-        m_tabWidget->tabBar()->setEnabled(false);
-        vLayout->addWidget(m_tabWidget);
+        QHBoxLayout* hLayout = new QHBoxLayout(this);
+        hLayout->setSpacing(0);
+        hLayout->setContentsMargins(0, 0, 0, 0);
+
+        QFrame* lhsFrame = CreateLHSTabs();
+        hLayout->addWidget(lhsFrame);
 
+        QFrame* rhsFrame = CreateRHSPane();
+        hLayout->addWidget(rhsFrame);
+        
+        QFrame* createGemFrame = new QFrame(this);
```

Comment:
FWIW you generally don't need to specify a parent in the constructor when you are going to add that widget to a layout using `addWidget` because it will call `SetParent` on that widget at that time which is usually enough to take care of most QWidget needs to ensure object deletion.